virttest/guest_agent.py

```diff
@@ -678,22 +678,35 @@ def verify_fsfreeze_status(self, expected):
             raise VAgentFreezeStatusError(self.vm.name, status, expected)
 
     @error_context.context_aware
-    def fsfreeze(self, check_status=True, timeout=FSFREEZE_TIMEOUT):
+    def fsfreeze(self, check_status=True, timeout=FSFREEZE_TIMEOUT,
+                 fsfreeze_list=False, mountpoints=None):
         """
-        Freeze File system on guest.
+        Freeze File system on guest, there are two commands,
+        "guest-fsfreeze-freeze" and "guest-fsfreeze-freeze-list",
+         the second one can freeze some of mountpoints.
```

Comment:
'the second one allow to freeze by mount points'?

Reply:
Updated.